Apps/phjira/jira_connector.py

```diff
@@ -1813,7 +1813,7 @@ def _handle_link_tickets(self, param):
         try:
             UnicodeDammit(link_type).unicode_markup.encode('utf-8')
         except:
-            return action_result.set_status(phantom.APP_ERROR, "Parameter value failed validation. Enter a valid value.")
+            return action_result.set_status(phantom.APP_ERROR, "'link_type' action parameter value failed validation. Enter a valid value")
```

Comment:
Would be helpful to suggest what type of value is expected

Reply:
Here, the data type of parameter is String. So, there is no value list or pre-defined format for value. We are just checking that if there are any invalid Unicode chars, we should raise an error. Although, if you like to improve the error message, you can suggest the error message as per the scenario. Thanks!!